examples/networkpolicy/kubernetes-cilium.json

```diff
@@ -0,0 +1,50 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.9",
+      "kubernetesConfig": {
+        "networkPolicy": "cilium",
+        "etcdVersion": "3.1.10",
```

Comment:
out of curiosity is there an etcd version dependency for cilium? current default in acs-engine is `3.2.16`.